filter/slice.go

```diff
@@ -0,0 +1,82 @@
+package filter
+
+import (
+	"fmt"
+
+	"github.com/AdRoll/baker"
+)
+
+var SliceDesc = baker.FilterDesc{
+	Name:   "Slice",
+	New:    NewSlice,
+	Config: &SliceConfig{},
+	Help:   "Slices the source field value to the given bytes length and start index and saves the value to the destination field. If the start index is greater than the field value lenght, set an empty string to destination",
```

Comment:
Both in python and in Go, slice expression is made of `[low:high]` where:
 - low and high are both indices
 - both can be omitted, which result in low being 0 and high being the index of the last element

There's no parameter that directly represent the length and IMHO we should follow this philosophy with the Slice filter.

Reply:
ok